optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -456,7 +456,7 @@ def _compile(self):
         if (
             "CACHE_DIR" not in ov_config.keys()
             and not str(self.parent_model.model_save_dir).startswith(gettempdir())
-            and self._device.lower() == "gpu"
+            and self._device.lower().split(":")[0] == "gpu"
```

Comment:
what about replacing it with for cases like device is set to "GPU.0" or "AUTO:GPU,CPU"
```suggestion
            and "gpu" in self._device.lower()
```